README.md

```diff
@@ -30,6 +30,7 @@ Available transforms:
 
 - [Shadow DOM to Light DOM](#shadow-dom-to-light-dom) (`shadow-to-light`)
 - [Synthetic Shadow DOM to Native Shadow DOM](#synthetic-shadow-dom-to-native-shadow-dom) (`synthetic-to-native`)
+- [HTML parsing errors](#html-parsing-errors)
```

Comment:
I think potentially we can put all template cleanup functions into a single codemod. They all have the same goal: remove warnings/errors in `.html` files.

In the context of an IDE for instance, the scenario would probably be:

1. User is editing an `.html` file
2. The IDE suggests you run a codemod on it

Plus, if we add codemods in the future for .js and .css files, then there are only three total, which seems simple to run on a whole codebase.